### DISCUSSION TITLE
Welcome to OOP-using-Java Discussions!

### DISCUSSION THREAD

**Discussion by mssanamateen** [mssanamateen/OOP-using-Java#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mssanamateen** [comment]
Hello All, 
Please feel free to share your ideas on Java and it's features

**Comment by SanaMateen** [comment]
Hello 
This is Java Enthusiast.Exploring new features of Java.
